packages/p2p/src/components/buy-sell/buy-sell-modal.scss

```diff
@@ -39,6 +39,36 @@
         }
     }
 
+    &--input {
+        display: flex;
+        gap: 2rem;
+        flex-direction: column;
+        .dc-input__wrapper {
+            margin-bottom: unset;
+        }
+
+        &-field {
+            display: flex;
+            gap: 2rem;
+        }
+    }
+
+    &-hintbox {
+        margin: -2.4rem 0 2.4rem;
+
+        @include mobile {
+            margin: 0 0 2.4rem;
+        }
+
+        .dc-hint-box {
+            &__icon {
+                align-self: flex-start;
+                min-height: 16px;
+                min-width: 16px;
```

Comment:
@likhith-deriv can we user rem's instead of px's here?

Reply:
Yes. I will change them to rem

Reply:
@likhith-deriv  `<Hintbox/>` icon default size is 16. Is this required here?

Reply:
No. I will remove this styling